core/src/main/scala/org/apache/spark/scheduler/sparrow/SparrowSchedulerBackend.scala

```diff
@@ -217,6 +217,8 @@ class SparrowSchedulerBackend(scheduler: SparrowScheduler)
     }
 
     driverEndpoint = createDriverEndpointRef(properties)
+    println("Waiting to Start the Backend - Press enter after done")
```

Comment:
Easier way for me to startup the backends, than having a fixes 5 or 10 secs to react within